src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -1,8 +1,38 @@
 package edu.stanford.dlss.was;
 
-public class WasapiDownloader
-{
-    public static void main(String[] args) {
-        System.out.println("I am WasapiDownloader. Hear me roar. Woo.");
+import java.io.*;
```

Comment:
Are we cool with importing entire packages? It could potentially pollute the namespace and make it harder to find out what class is being used. Google, Mozilla and Twitter all explicitly disallow wildcard imports:

https://github.com/twitter/commons/blob/master/src/java/com/twitter/common/styleguide.md#no-wildcard-imports

https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#Java_practices

https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports